recipe/meta.yaml

```diff
@@ -1,45 +1,43 @@
 {% set name = "lbzip2" %}
 {% set version = "2.5" %}
-{% set sha256 = "eec4ff08376090494fa3710649b73e5412c3687b4b9b758c93f73aa7be27555b" %}
+{% set sha256 = "7be69ece83ecdc8f12b9201d838eee5cdb499f2fd68cffd2af58866076ccac43" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
-# UNCOMMENT AFTER THIS ISSUE IS SOLVED:
-# https://github.com/conda-forge/docker-images/pull/36
-# source:
-#   fn: lbzip2-{{ version }}.tar.bz2
-#   url: http://archive.lbzip2.org/lbzip2-{{ version }}.tar.bz2
-#   sha256: {{ sha256 }}
+source:
+  url: https://github.com/kjn/lbzip2/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+     # from gnulib-devel srpms in fedora 42 - gnulib-devel-0-52.20230709git.fc42.noarch.rpm
+  patches:   
+    - lbzip2-c99.patch
+    - lbzip2-gnulib.patch
 
 build:
-  number: 0
-  skip: True  # [win]
-  script:
-    - curl http://archive.lbzip2.org/lbzip2-{{ version }}.tar.bz2 -o lbzip2-{{ version }}.tar.bz2
-    - openssl sha256 lbzip2-{{ version }}.tar.bz2 | grep {{ sha256 }}
-    - tar xjvf lbzip2-{{ version }}.tar.bz2 --no-same-owner
-    - cd lbzip2-{{ version }}
-    - ./configure --prefix=$PREFIX
-    - make -j${CPU_COUNT}
-    - make check
-    - make install
+  number: 1
+  skip: true  # [win]
 
 requirements:
   build:
-    - toolchain
```

Comment:
it's been quite a while since I stumbled over a recipe still stuck at the old `toolchain`. Congrats for successful necromancy here 🧟 😉